src/pipelines/sass.rs

```diff
@@ -30,11 +32,16 @@ impl Sass {
         // Build the path to the target asset.
         let href_attr = attrs
             .get(ATTR_HREF)
-            .context(r#"required attr `href` missing for <link data-trunk rel="sass|scss" .../> element"#)?;
+            .context(r#"required attr `href` missing for <link data-trunk [data-inline] rel="sass|scss" .../> element"#)?;
```

Comment:
Let's actually remove the added `[data-inline] ` bit, as that attribute, along with any of the other possible attributes are covered by the `...`.

Reply:
I just removed `[data-inline]`. Is this correct? @torhovland 

I'm asking because you said:
> along with any of the other possible attributes are covered by the `...`

Reply:
Hmm, not sure why you're asking me...

Reply:
OK, no problem.